src/pathrewriter.ts

```diff
@@ -24,15 +24,18 @@ import constants from './constants';
 
 const pathPosix = path.posix;
 
-class PathResolver {
-  constructor(public abspath?: boolean) {
+class PathRewriter {
```

Comment:
Moved to module of functions called urlUtils.  The actual ast rewriter function is moved to Bundler.